WalletWasabi.Tests/UnitTests/WabiSabi/Client/ArenaClientTests.cs

```diff
@@ -203,7 +204,7 @@ public async Task SignTransactionAsync()
 			using Arena arena = await WabiSabiFactory.CreateAndStartArenaAsync(config, round);
 
 			var mockRpc = new Mock<IRPCClient>();
-			await using var coordinator = new ArenaRequestHandler(config, new Prison(), arena, mockRpc.Object);
+			await using var coordinator = new ArenaRequestHandler(config, new Prison(), arena);
```

Comment:
Why the variable is called `coordinator`?

Reply:
Because `ArenaRequestHandler` implements the coordination so, it is the coordinator. This is problematic because the `WabisabiCoordinator` sound like the real coordinator but that is really a class to wire components, something that typically is done by the DI container.